### ISSUE TITLE
Reject port 80 non-onion destinations

### ISSUE THREAD

**Issue by JeremyRand** [namecoin/StemNS#21]
We should add an option (per eTLD) to reject destinations that are not `.onion` domains if they are using TCP port 80.  This has the effect of guaranteeing that the eTLD will use a secure origin for HTTP traffic.

**Comment by JeremyRand** [comment]
Maybe wait on this until Namecoin TLS support for Tor Browser is further along?

**Comment by yanmaani** [comment]
Seems a bit gross. If it's needed, it's needed, but it seems conceptually cleaner to limit ourselves to `.onion` and punt on this until later.

**Comment by JeremyRand** [comment]
Seems low priority at the moment, but if a volunteer wants to implement this, I'd accept a PR.
